tests/auth/test_github.py

```diff
@@ -173,14 +176,25 @@ def test_config_schema_empty_cache() -> None:
         _config = gh.Config.from_dict(options)
 
 
+def test_config_schema_api_timeout() -> None:
+    with pytest.raises(ValidationError):
+        _config = gh.Config.from_dict({"api_timeout": "invalid"})
+    cfg = gh.Config.from_dict({"api_timeout": 1})
+    assert cfg.api_timeout == 1.0
+    cfg = gh.Config.from_dict({"api_timeout": [1, 2]})
+    assert cfg.api_timeout == (1.0, 2.0)
+    with pytest.raises(ValidationError):
+        _config = gh.Config.from_dict({"api_timeout": [1, "invalid"]})
+
+
 DEFAULT_CONFIG = gh.Config.from_dict({})
-DEFAULT_TOKEN_DICT = {
+DEFAULT_USER_DICT = {
     "login": "kingofthebritons",
     "id": "12345678",
```

Comment:
Given the example, I'd suggest the id be `125678` but....

Reply:
Heh, I couldn't figure out why you suggest that :thinking:, care to share? But I'm doing it, as it seems it could be some funny reference or what.